src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -123,66 +126,72 @@ private void writeMapToJSON(
 		for (String key : data.keySet()) {
 			Object value = data.get(key);
 
-			MaxwellJson.writeValueToJSON(g, includeNullField, key, value);
+			MaxwellJson.writeValueToJSON(g, includeNullField, applyNamingStrategy(key), value);
 		}
 
 		g.writeEndObject(); // end of 'jsonMapName: { }'
 	}
 
+	private String applyNamingStrategy(String name) {
+		return this.fieldNameStrategy.apply(name);
+	}
+
 	public String toJSON() throws Exception {
 		return toJSON(new MaxwellOutputConfig());
 	}
 
 	public String toJSON(MaxwellOutputConfig outputConfig) throws Exception {
+		this.fieldNameStrategy = new FieldNameStrategy(outputConfig.namingStrategy);
```

Comment:
sorry I didn't notice the first time through: there's no need for this to be an instance variable.  Just make it a local variable please (and pass it into `writeMapToJSON`)

Reply:
yes, there's no need to declare an instance variable. i've updated the code.